.github/workflows/create-tag.yml

```diff
@@ -0,0 +1,106 @@
+name: "Create a tag"
+
+on:
+  workflow_call:
+    inputs:
+      ref:
+        description: "Ref to be tagged"
+        required: true
+        type: string
+        default: master
+      tag:
+        description: "Tag for new version (v1.23.4)"
+        required: true
+        type: string
+      create_release:
```

Comment:
Part of the problem is that we tag without creating a release, this shouldn't be optional.